### ISSUE TITLE
[Feature Request] :Adding a login or signup option

### ISSUE THREAD

**Issue by Kumud14** [GameSphere-MultiPlayer/GameSphere#930]
## Connect Us On Social Media -
- [ ]  Star the repositry
- [ ]  <a href="https://github.com/GameSphere-MultiPlayer" target="_blank">GitHub</a>
- [ ]   <a href="https://www.linkedin.com/company/gamesphere-multiplayer/" target="_blank">LinkedIn</a>
- [ ]   <a href="https://www.youtube.com/@durgesh4993/channels" target="_blank">YouTube</a>
- [ ]    <a href="https://discord.com/invite/dakzwdz4ev" target="_blank">Discord</a>

**Comment by Aiko002** [comment]
@Durgesh4993 I would like to work on this issue. kindly assign it to me

**Comment by Durgesh4993** [comment]
Could you please explain more

**Comment by Aiko002** [comment]
@Durgesh4993 This says that there is no option for login or signup buttons currently. I can add those as required

**Comment by Kumud14** [comment]
> Could you please explain more

yes .. so just after loading..part, we can add this feature with a welcome note( or this option just above the settings icon can be added ). Users have to log in or sign up with their email ID, and another option like a forgot password would exist. For exploring the website one time sign in is a must
